Emby.Server.Implementations/Library/CoreResolutionIgnoreRule.cs

```diff
@@ -60,23 +32,15 @@ public bool ShouldIgnore(FileSystemMetadata fileInfo, BaseItem parent)
                 return false;
             }
 
-            var filename = fileInfo.Name;
-
-            // Ignore hidden files on UNIX
-            if (Environment.OSVersion.Platform != PlatformID.Win32NT
-                && filename[0] == '.')
```

Comment:
this check should be retained I think. I propose to replace MacOS-specific ignore with it

Reply:
We should not have platform specific ignores. Problem is someone uses say NFS or Samba to mount a volume on macOS and it generates the special files. Now plex running on Win/Linux will see these files. And vice versa for thumbs.db etc. Could we ignore all dotfiles?

Reply:
That's what I was suggesting right below, to replace MacOS-specific ignore with Unix hidden files' ignore.

Reply:
I misunderstood you wanted ignores to vary by platforms.

Reply:
@JustAMan Ok, I've addressed your concerns. I changed it to ignore all unix hidden files **and** directories.